src/streamlink/compat.py

```diff
@@ -55,8 +55,8 @@ def bytes(b, enc="ascii"):
 
 
 
-getargspec = getattr(inspect, "getfullargspec", inspect.getargspec)
 
+getargspec = getattr(inspect, "getfullargspec", inspect.getargspec)
```

Comment:
you did not change anything here, so it should not be in the history

Reply:
How can I do that?

Reply:
Make a commit to undo the change, then [squash your commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing).

Reply:
Sorry, I've 'Zero Bock' for long attempts, do you have detailed instructions for that?

Reply:
Just commit the change to revert the change and we'll squash it at merge time :)